packages/babel/package.json

```diff
@@ -58,6 +60,7 @@
     "@babel/plugin-transform-runtime": "^7.7.4",
     "@babel/preset-env": "^7.9.0",
     "@rollup/plugin-json": "^4.0.0",
+    "@types/babel__core": "^7.1.8",
```

Comment:
according to Maël Nison (Yarn2 maintainer) this should be also be specified as optional peer dependency, which can be done by adding this to package.json:
```
  "peerDependenciesMeta": {
    "@types/babel__core": {
      "optional": true
    }
  },
```

This might create some usability issues though as one doesn't necessarily will have those types installed and thus our types for `@rollup/plugin-babel` won't work. Maybe I'm worried about this prematurely though, I'm ok with shipping this (with this optional peer dep specified) and just seeing what happens.

Reply:
Annoyingly this makes the typings for the `test:ts` run become `any`. Hence the tests are passing, even though Ideally we need https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45613

Reply:
At least for local tests, we could add a devDependency too.

Reply:
Good idea 👍